site/content/docs/10-exporting.md

```diff
@@ -42,7 +42,7 @@ You can also add a script to your package.json...
 
 ### How it works
 
-When you run `sapper export`, Sapper first builds a production version of your app, as though you had run `sapper build`, and copies the contents of your `static` folder to the destination. It then starts the server, and navigates to the root of your app. From there, it follows any `<a>` elements it finds, and captures any data served by the app.
+When you run `sapper export`, Sapper first builds a production version of your app, as though you had run `sapper build`, and copies the contents of your `static` folder to the destination. It then starts the server, and navigates to the root of your app. From there, it follows any `<a>`, `<img>` and `<source>` elements it finds, and captures any data served by the app.
```

Comment:
Tbh, I'm quite confused with this addition. Does it mean externally hosted images (e.g. links from cdn or s3) will be captured and stored locally in export directory? Also, with the addition of `<source>`, does it means it works with video formats too, i.e. mp4, webm, ogg?

Reply:
No, as with `<a>`, only things that resolve to local paths served by your app are crawled and stored. For `<source>`, this refers only to `srcset` attributes. I'm definitely open to having the docs here explain this in more depth.

Reply:
It's probably a good clarification to add that it only crawls the local images

Reply:
Local images meaning contents from `static`? If so, doesn't sapper already copied the whole folder over?

Reply:
> For `<source>`, this refers only to `srcset` attributes.

Ah, I'm assuming this refers to `srcset` in `<source>` that is a direct descendant from `<picture>`, citing from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source#attr-srcset then, what about something like this?

```suggestion
When you run `sapper export`, Sapper first builds a production version of your app, as though you had run `sapper build`, and copies the contents of your `static` folder to the destination. It then starts the server, and navigates to the root of your app. From there, it follows any `<a>`, `<img>` and `<source>` (only for a direct child of `<picture>` element) elements it finds, and captures any data served by the app.
```

If we were to clarify `<img>` too, perhaps it's better to separate them in another paragraph.